controllers/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -2067,3 +2072,84 @@ def _test_delete_host_rmhost_errors(self, client_error, expected_error):
 
     def test_delete_host_errors(self):
         self._test_delete_host_rmhost_errors(Exception("Failed"), Exception)
+
+    def _mock_cli_volume_group(self, uid="", volume_count=0):
+        volume_group = {svc_settings.VOLUME_GROUP_VOLUME_COUNT_ATTR_KEY: str(volume_count),
+                        svc_settings.VOLUME_GROUP_NAME_ATTR_KEY: common_settings.VOLUME_GROUP_NAME,
+                        svc_settings.VOLUME_GROUP_ATTR_KEY: common_settings.INTERNAL_VOLUME_GROUP_ID}
+        if uid:
+            volume_group["uid"] = uid
+
+        return Munch(volume_group)
+
+    def _prepare_lsvolumegroup(self, no_return=None, volume_count=0):
+        if no_return:
+            return_value = None
+        else:
+            return_value = self._mock_cli_volume_group(volume_count=volume_count)
+        self.svc.client.svcinfo.lsvolumegroup.return_value = Mock(as_single_element=return_value)
+
+    def test_create_volume_group(self):
```

Comment:
can you add `success` and `failed` prefixes to the test functions?

Reply:
done